pkg/api/v1/types.go

```diff
@@ -684,6 +685,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,4,opt,name=readOnly"`
 }
 
+// Represents a Quobyte mount that lasts the lifetime of a pod.
+// Quobyte volumes do not support ownership management or SELinux relabeling.
+type QuobyteVolumeSource struct {
+	// Registry represents a single or multiple Quobyte Registry services
```

Comment:
This doesn't tell me what sort of value goes here.  Is it a random number?  The name of a Kubernetes Service?  A URL?  Some key from a different API?